lib/run-compilation.js

```diff
@@ -26,8 +26,8 @@ module.exports = function runCompilation (options, compiler, done) {
         return file.errored;
       });
 
-      if (!options.quiet) {
-        console.log(chalk.yellow(options.formatter(results)));
+      if (options.quiet === false) {
```

Comment:
Hey @JaKXz, I tested this locally. 
It seems like this change has no effect, because `quite` is set default to false [here](https://github.com/JaKXz/stylelint-webpack-plugin/blob/master/index.js#L19).
Wonder what is the reason of using `quite` here if webpack outputs warnings and errors to the console as well on `onCompilation`?
Maybe we could move it to `onCompilation` and have something like this:
```js
compiler.plugin('after-emit', function onCompilation (compilation, callback) {
    if (warnings.length) {
      if (options.quiet === false) {
        compilation.warnings.push(chalk.yellow(options.formatter(warnings)));
      }
      warnings = [];
    }

    if (errors.length) {
      if (options.quiet === false) {
        compilation.errors.push(chalk.red(options.formatter(errors)));
      }
      errors = [];
    }

    callback();
  });
```

If at least something should be in the output, it can be:
```js
if (warnings.length) {
  var formattedWarnings = chalk.yellow(options.formatter(warnings));
  if (options.quiet === false) {
    compilation.warnings.push(formattedWarnings);
  } else {
    console.log(formattedWarnings);
  }
  warnings = [];
}
```

what do you think?